### PR TITLE
[14.0][IMP] Add operations to the new BOM created by the configurator (When the Master BOM has Operations)

### PR DESCRIPTION
**Pull request by Vimal-OSI**

### CODE REVIEW

File: product_configurator_mrp/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Product Configurator Manufacturing",
-    "version": "14.0.1.2.0",
+    "version": "14.0.1.2.1",
```
Comment (dreispt) [inline product_configurator_mrp/__manifest__.py]: No need to bump version, the merge bot will do that for you.

File: product_configurator_mrp/tests/test_mrp.py [hunk 2]
```diff
@@ -109,7 +119,7 @@ def test_01_action_config_start(self):
             context
         )
         self._configure_product_nxt_step()
-        # self.assertEqual(
+
```
Comment (dreispt) [inline product_configurator_mrp/tests/test_mrp.py]: Why delete only the first line of the commented statement?

File: product_configurator_mrp/tests/test_mrp.py [hunk 3]
```diff
@@ -37,6 +39,14 @@ def setUp(self):
...
             }
         )
+        # create BOM operations line
+        self.mrpRoutingWorkcenter.create(
+            {
+                "bom_id": self.bom_id.id,
+                "name": "Operation 1",
+                "workcenter_id": self.workcenter_id.id,
+            }
+        )
```
Comment (dreispt) [inline product_configurator_mrp/tests/test_mrp.py:49]: Kudos for touching the tests!
Now if we could only add the check that the operations were really copied, that would be the cherry on top of the cake.

**Review by dreispt — APPROVED** [review]

File: product_configurator_mrp/tests/test_mrp.py (hunk 3 above)
Comment (Vimal-OSI) [reply product_configurator_mrp/tests/test_mrp.py:49]: @dreispt , Yeah but testcase file is on commented.

https://github.com/OCA/product-configurator/blob/146ce493130408d5f7fa8112eaa4c65749751226/product_configurator_mrp/tests/__init__.py#L4

https://github.com/OCA/product-configurator/blob/146ce493130408d5f7fa8112eaa4c65749751226/product_configurator/tests/__init__.py#L1

File: product_configurator_mrp/__manifest__.py (hunk 1 above)
Comment (Vimal-OSI) [reply product_configurator_mrp/__manifest__.py]: Done the changes

**Pull request merged**